distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -15,20 +18,23 @@ function formatData(item) {
     result.push({
       x: data[i]['refRank'] || data[i]['b'],
       y: data[i]['c'] * 100,
+      p: data[i]['p'] * 100,
```

Comment:
That's a good question for @robhudson since I believe this is what he said unless I misunderstood.